check/configuration/supported/clang-format.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Clang Format
+---
+
+# ClangFormat
+
+ClangFormat is a formatter for C.
```

Comment:
Should be C, C++, see https://github.com/trunk-io/docs/pull/239#discussion_r1511619385

Reply:
Still think this one should be changed